pkg/store/storepb/rpc.proto

```diff
@@ -157,6 +143,33 @@ message LabelValuesRequest {
 }
 
 message LabelValuesResponse {
-  repeated string values = 1;
+  repeated string values   = 1;
   repeated string warnings = 2;
 }
+
+/// Rules represents API that is responsible for gathering rules and their statuses.
+service Rules {
+  /// Rules has info for all rules.
+  rpc Rules(RulesRequest) returns (stream RulesResponse);
+}
+
+message RulesRequest {
+  enum Type {
+    ALL       = 0;
```

Comment:
I feel like it would be more logical to have `ALL` defined as `3` so that `ALERTING|RECORDING` bitmap operation would be equal to `ALL` i.e. `3`.

Reply:
Disagree, as `0` is also the default if not set.

Reply:
And default is ALL ):